modules/cnf-about-topology-aware-scheduling.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-topology-aware-scheduling.adoc
+
+[id="cnf-about-topology-aware-scheduling_{context}"]
+= About topology-aware scheduling
+
+NUMA architecture allows a CPU to use any available compute memory regardless of where the memory is located. This allows for increased flexibility at the expense of performance. A CPU processing a workload using memory that is outside its NUMA zone is slower that a workload processed in a single NUMA zone. Also, for I/O-constrained workloads, the network interface on a distant NUMA zone slows down how quickly information can reach the application. High-performance workloads, such as telco workloads, can’t operate to specification under these conditions. To process latency-sensitive or high-performance workloads as efficiently and quickly as possible, all the requested {product-title} compute resources (CPUs, memory, devices) should be aligned in the same NUMA zone.
```

Comment:
"NUMA architecture allows a CPU to use any available compute memory regardless of where the memory is located."
I'd add a clarification that this applies when we have multiple memory controllers in the system, related to different CPUs complexes. Albeit important, we are in the space of the clarifications.

Reply:
@fromanirh What about: "NUMA architecture allows a CPU to use any available memory across CPU complexes regardless of where the memory is located. This applies when there are multiple memory controllers in the CPU."